python/lancedb/query.py

```diff
@@ -490,7 +508,11 @@ def to_arrow(self) -> pa.Table:
         # open the index
         index = tantivy.Index.open(index_path)
         # get the scores and doc ids
-        row_ids, scores = search_index(index, self._query, self._limit)
+        query = self._query
+        if self._phrase_query:
+            query = query.replace('"', "'")
```

Comment:
Do we need to worry about escaped double quotes at all?

Reply:
i added a test case with escaped double quotes. It does not cause a syntax error in this case